.github/pull_request_template.md

```diff
@@ -1,23 +1,20 @@
-## Related Issues
-<!-- Link issues that this PR might resolve/fix. If an issue doesn't exist, include a brief motivation for the change being made -->
+## Related Issues & Context
+<!-- Link issues that this PR might resolve/fix. If an issue doesn't exist, including the context for the change being made -->
 
 ## Proposed Changes
 <!-- A clear list of the changes being made -->
 
-## Additional Info
-<!-- Callouts, links to documentation, and etc -->
+## Test Plan
+<!-- PRs should include unit/integration tests, and/or IRL opertional testing plans-->
 
 ## Checklist
 
-Before you mark the PR ready for review, please make sure that:
+@lotus-maintainers will only approve the PR if all boxes below are checked:
 
 - [ ] Commits have a clear commit message.
 - [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
   - example: ` fix: mempool: Introduce a cache for valid signatures`
   - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
   - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
-- [ ] New features have usage guidelines and / or documentation updates in
-  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
-  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
-- [ ] Tests exist for new functionality or change in behavior
+- [ ] New or updated features/configuration have usage guidelines and/or documentation updates in [Lotus Documentation](https://lotus.filecoin.io).
```

Comment:
In practice, I feel like we don't do this, and are unlikely to start doing this. What do we think about updating this to say that the PR description itself should have documentation guidelines (we often, but not always do this today), and perhaps we should add such docs to the UNRELEASED section of the Changelog as needed?

Reply:
I think I agree to this. We should at least require that PR includes the details about new feature/configuration and that preferably a test plan that demonstrates the usage. Updating UNRELEASED in the lotus should help capture the bigger items which can then collected and capture together in the lotus.filecoin.io docs.